docs/index.rst

```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
 This installs the executables ``pathfinding-service``, ``monitoring-service``
 and ``request-collector`` into the virtual environment.
 
+Using MS and PFS in raiden
```

Comment:
While this is nice, doesn't this belong into the client docs?

Reply:
Yes this does belong to the client about to remove it.